packages/bot-web-ui/src/stores/journal-store.js

```diff
@@ -29,8 +36,14 @@ export default class JournalStore {
         { id: message_types.SUCCESS, label: localize('System') },
     ];
 
+    getJournalStorageKey = key => {
```

Comment:
We're not getting the key here, but the actual object saved to our `sessionStorage`. Try to name the function accordingly, e.g. `getJournalFromSessionStorage`.